File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 0
        counter = 0
        while i < len(nums):
            counter += 1
            try:
                while nums[i] == nums[i + 1]:
                    nums.pop(i + 1)
            except IndexError:
                break
            i += 1
        return counter

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int index1 = 1; // Represent true Index
        int lastArrayVal = nums[0]; // For Control Duplicates

        for (int i = 1; i < nums.length; i++) {
            if (nums[i] != lastArrayVal) {
                nums[index1] = nums[i];
                index1++;
                lastArrayVal = nums[i];
            }
        }
        return index1;
    }
}